### PR TITLE
Modify h2, h3, h4 headers for search.

### PR DESCRIPTION
**Pull request by johnnymatthews**
Went through all the pages _except_ everything in `reference`. Since that stuff is automated-ish anyway it doesn't make sense to manually go through and modify headers to fit the search function we currently have.

I also cleaned up some weird formatting and code blocks where I saw them, but this is by no means a thorough inspection and fix of all the docs. There's still a bunch of things wrong that I want to fix, but probably in the new year.

### CODE REVIEW

**Comment by johnnymatthews** [comment]
I **haven't updated** `.vuepress/config.js` to match the changes in headers in this merge.

**Comment by jessicaschilling** [comment]
@johnnymatthews You shouldn't have to change the config if you're just talking about wanting H2s to update; those happen automatically. Are you wanting to change other headers/titles?

File: docs/how-to/command-line-quick-start.md [hunk 1]
```diff
@@ -84,11 +84,15 @@ If you want, you can also build IPFS from source. For more detailed instructions
 ::: tip
 **Already have IPFS installed, but need to upgrade?** Note that upgrades (and downgrades) may involve a repository upgrade process performed by the [fs-repo-migrations](https://dist.ipfs.io/#fs-repo-migrations) tool.
 
-#### Upgrading using ipfs-update
+#### Updade IPFS
```
[the remark this answers is not in the document]
Comment (johnnymatthews) [reply docs/how-to/command-line-quick-start.md]: Not sure how our search function works. If it's separating the words with the `/` then it's ok, but if it's reading `Update/upgrade` at one single word we might be in trouble.

[the remark this answers is not in the document]
Comment (johnnymatthews) [reply docs/how-to/command-line-quick-start.md]: Also just realized I spelt `Upgrade` `Updade`. How embarrassing...

**Comment by johnnymatthews** [comment]
> Thanks for tackling this (especially the command-line stuff) ... I suggested some edits for consistency, but otherwise LGTM. Do you have sufficient permissions to merge?

Yeahp, looks like I can merge straight into the repo.

File: docs/how-to/command-line-quick-start.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jessicaschilling) [reply docs/how-to/command-line-quick-start.md]: Ah, I thought you actually meant `update`. Let's just go with `Upgrade IPFS`?

**Pull request merged**